samples/winapi_drivelist/main.c

```diff
@@ -56,10 +67,32 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
+    debugPrint("\nDone! Exiting in 10 seconds...");
+
+    Sleep(10000);
+
+    ret = nxUnmountDrive('C');
+    // If there was an error while unmounting (nxUnmountDrive() returns false)
+    if (!ret) {
+      error = GetLastError();
+      errorCount++;
+      debugPrint("Couldn't unmount C: drive Reason: %s\n. Trying to unmount E: drive...\n", error);
+      // Leave some time for the user to read the error
+      Sleep(5000);
```

Comment:
You already wait 10 seconds if any error happened. Waiting an additional 5 seconds per error is not necessary in my opinion.

*(Other instances, too)*